### ISSUE TITLE
For Combobox with autocomplete = 'on', selected item is not narrated by Windows Narrator

### ISSUE THREAD

**Issue by SarveshNamjoshi** [microsoft/fluentui#12498]
**Environment Information**
         Package version(s): current
         Browser and OS versions: Microsoft Edge Version 81.0.416.34 (Official build) beta (64-bit)

**Please provide a reproduction of the bug in a codepen:**
https://codepen.io/Namjoshi/pen/oNXVmNp
repro-steps : 1. Run the codepen in Edge (preferably in Edge chromium)
              2. Start windows narrator (with scan mode off)
			  3. navigate till combobox
			  4. Type 'w' in the combobox ('Watermelon' is automatically selected)
			  4. Observe that narrator is not narrating the selected item 'Watermelon'.
			  
**Actual behavior:**
For Combobox with autocomplete = 'on', windows narrator is not narrating the selected item.

**Expected behavior:**
Narrator should narrate the selected item in the combobox.

**Priorities and help requested:**
Requested priority: High

**Comment by aneeshack4** [comment]
@SarveshNamjoshi Thanks for reporting this - I confirm repro:
![image](https://user-images.githubusercontent.com/4161791/78207743-66915a00-7457-11ea-8bd0-119b955fdad3.png)

It looks like the value is correctly changing when I inspect the DOM: 
![image](https://user-images.githubusercontent.com/4161791/78207965-06e77e80-7458-11ea-9a3a-f65c22ea9414.png)
And the other aria roles, like `aria-autocomplete` that need to be set for combobox according to the [W3 spec](https://w3.org/TR/wai-aria-1.1/#combobox) seem to be properly defined/changing. So I'm not sure what could be causing this - @joschect any ideas?

**Comment by SarveshNamjoshi** [comment]
@aneeshack4,
Thanks for the confirmation about repro. Could you please provide any ETA for the fix.

**Comment by joschect** [comment]
@aneeshack4 I'll need to take a look. I suspect that this is an Edge Chromium problem but I am not sure. Per your comment all of the expected aria attributes should be present.

**Comment by SarveshNamjoshi** [comment]
@joschect any updates on this bug?

**Comment by joschect** [comment]
@SarveshNamjoshi I took a look and it looks like you're using Fabric 6, is that the case? There have been a lot of fixes and updates to the combobox, especially regarding accessibility. 

That being said I'm noticing a similar problem in Fabric 7, although I can confirm that the appropriate value is in the input element. I am not sure it's not being read out and I'll have to investigate a bit more.

**Comment by SarveshNamjoshi** [comment]
@joschect , in my code pen I have used Fabric 6 but I am facing this issue even in **Fabric 7** as well.
Sure, you investigate on this and guide me acordingly. thanks.

**Comment by joschect** [comment]
@SarveshNamjoshi Yep! I mentioned in my previous reply that I saw the same problem in fabric 7 and would need to investigate.

**Comment by SarveshNamjoshi** [comment]
@joschect just wondering if you got chance to investigate and you have further updates on this bug.

**Comment by joschect** [comment]
@SarveshNamjoshi I haven't had a chance to investigate. This is definitely a bug and we'll get it looked at.

**Comment by aneeshack4** [comment]
@SarveshNamjoshi We're currently focusing on improvements to performance and coherence, as you can see in the pinned issue on our repo #12770 detailing our v8 plans. And since this isn't a breaking issue right now, I'm going to put won't fix on this and close it. While we don't plan to address this right now, when we get a chance to work on ComboBox in the future, this will be taken into consideration. Thank you for reaching out to us!
